### PR TITLE
M3-1453: Add tags to nodeBalancers landing page

### PR DESCRIPTION
**Pull request by rulikrulit**
- Adds tags display to nodeBalancers table on the landing page
- Adds missing tags variable check to tags component

### CODE REVIEW

File: src/components/Tags/Tags.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ export class Tags extends React.Component<CombinedProps, {}> {
 
   render() {
     const { tags } = this.props;
+    if (!tags) return null;
```
Comment (Jskobos) [inline src/components/Tags/Tags.tsx]: Linter is complaining that the if-statement isn't bracketed

File: src/types/NodeBalancer.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ namespace Linode {
     updated: string;
     transfer: BalancerTransfer;
     config?: NodeBalancerConfig[];
+    tags?: string[];
```
Comment (martinmckenna) [inline src/types/NodeBalancer.ts]: when this is in production, the correct typing will be either an empty array or an array of strings, right?

We probably shouldn't type this an a possible undefined if the property is guaranteed to come back.

File: src/components/Tags/Tags.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import ShowMore from 'src/components/ShowMore';
 import Tag from 'src/components/Tag';
 
 export interface Props {
-  tags: string[];
+  tags: string[] | undefined;
```
Comment (Jskobos) [inline src/components/Tags/Tags.tsx]: Tags will always come back from the API as an array (empty if no tags). This doesn't affect the null checking you're doing below (which is safe and good), but TS should warn us if we ever try to pass `undefined` in here.

**Comment by rulikrulit** [comment]
At the moment there are no tags for NodeBalancers and without undefined check it leads to a critical error. I'd like to keep undefined check temporarily for the potential case where we will have the UI updated but there will be an old version of API (rollback, etc.). However I made a note for myself to clean it in future.

**Pull request merged**